tests/publiccloud/check_registercloudguest.pm

```diff
@@ -19,7 +19,9 @@ use strict;
 use utils;
 use publiccloud::utils;
 use publiccloud::ssh_interactive 'select_host_console';
-use version_utils 'is_sle';
+
+my $path = is_sle('=15-SP2') ? '/usr/sbin/' : '';    # 15-SP2 is the oldest version that needs fullpaths
+my $regcode_param = (is_byos()) ? "-r " . get_required_var('SCC_REGCODE') : '';
```

Comment:
Some notes, please:

1. These 2 lines of code are already performed in `sub registercloudguest`:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/d9fcc37e50f397fc08bd5a60ac0d3d6e05ec067e/lib/publiccloud/utils.pm#L110: those should be removed from here and function calls adapted to that.
We should eventually change code in that routine, accordingly.

2. Moreover, by using this contruction, when **NOT BYOS** all parameters are `empty`, but in the function calls it still is executed without parameters, like `sudo registercloudguest`: but `SCC_REGCODE` is mandatory in that routine, then would fail when missing.
Shouldn't we compose here, instead, the _full command_, eventually only `true` if-not-byos?